### PR TITLE
[PATCH v1] example: tm: destroy tm queues on example exit

### PR DESCRIPTION
**Pull request by muvarov**

### CODE REVIEW

**Comment by Bill-Fischofer-Linaro** [comment]
@bala-manoharan You may also wish to add your review verifying that this works on Cavium's implementation as well.

File: platform/linux-generic/odp_traffic_mngr.c [hunk 1]
```diff
@@ -3990,23 +3987,6 @@ int odp_tm_queue_destroy(odp_tm_queue_t tm_queue)
...
-	if (tm_wred_node != NULL) {
-		if (tm_wred_node->threshold_params != NULL)
-			return -1;
-
-		for (color = 0; color < ODP_NUM_PACKET_COLORS; color++) {
-			wred_params = tm_wred_node->wred_params[color];
-			if (wred_params != NULL)
-				return -1;
-		}
-	}
```
Comment (lumag) [inline platform/linux-generic/odp_traffic_mngr.c]: I don't get it, why are you removing this check?

Comment (lumag) [reply platform/linux-generic/odp_traffic_mngr.c]: ok

**Comment by lumag** [comment]
Reviewed-by: Dmitry Eremin-Solenikov

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**